airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -23,6 +23,8 @@
     DestinationType.MYSQL.value: 64,
     # https://oracle-base.com/articles/12c/long-identifiers-12cr2
     DestinationType.ORACLE.value: 128,
+    # https://docs.microsoft.com/en-us/sql/odbc/microsoft/column-name-limitations?view=sql-server-ver15
+    DestinationType.MSSQL.value: 64,
```

Comment:
wasn't the limit up to 128?

Reply:
Decided to as is (64 limit) because of support for older SQL Server versions, like 2014,  2016.